Gemfile

```diff
@@ -0,0 +1,54 @@
+source 'https://rubygems.org'
+
+git_source(:github) do |repo_name|
+  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
+  "https://github.com/#{repo_name}.git"
+end
+
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'rails', '~> 5.1.4'
+# Use postgresql as the database for Active Record
+gem 'pg', '~> 0.18'
+# Use Puma as the app server
+gem 'puma', '~> 3.7'
+# Use SCSS for stylesheets
+gem 'sass-rails', '~> 5.0'
+# Use Uglifier as compressor for JavaScript assets
+gem 'uglifier', '>= 1.3.0'
```

Comment:
About the gems: remember not to add dependencies that you aren't going to use. In this case since it's just the training it doesn't matter, but when you start a project by cloning the bootstrap go gem by gem and see if you need it. For example, if you aren't going to have views because it's a 100% API project, remove
```
# Use SCSS for stylesheets
 +gem 'sass-rails', '~> 5.0'
 +# Use Uglifier as compressor for JavaScript assets
 +gem 'uglifier', '>= 1.3.0'
```

and 
 
```
# Use CoffeeScript for .coffee assets and views
 +gem 'coffee-rails', '~> 4.2'
 +# Turbolinks makes navigating your web application faster. Read more: https://github.com/turbolinks/turbolinks
 +gem 'turbolinks', '~> 5'
```
 If you aren't going to write integration tests with browser automation, remove selenium, etc.

And trust me, the bootstrap has a lot more gems included by default so it's gonna be worse than this